install/kubernetes/cilium/charts/hubble-ui/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+You have successfully installed Cilium {{ .Chart.Name }}
+
+Your release version is {{ .Chart.Version }}
```

Comment:
This refers to the version of the Chart, right? Meaning, this currently would print something like `You have successfully installed Cilium hubble-ui. Your release version is 1.8.0`.

I think that's a bit confusing to users, since the deployed Hubble-UI version (currently v0.5) will always be independent of Cilium version number (currently 1.7.90).

Therefore, I think the message should be reformulated to make it clear that the version refers to the Cilium Chart version, and not to the Hubble UI version.

Reply:
Ok makes sense.

Reply:
Done